src/locales/en.json

```diff
@@ -285,12 +287,5 @@
     "passwordUpdated": "Password successfully updated",
     "updatedUserInfo": "Succesfully updated user info.",
     "userInfoUpdated": "User information successfully updated"
-  },
-  "contact": {
```

Comment:
I am not sure how it got removed, but the contact translation were quite useful...

Reply:
How did this happen lol, will fix

Reply:
Fixed!